apps/web/src/modules/dashboard/DaoAuctionCard.tsx

```diff
@@ -0,0 +1,159 @@
+import { Box, Flex, Text } from '@zoralabs/zord'
+import dayjs from 'dayjs'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import React, { useState } from 'react'
+import { formatEther } from 'viem'
+import { useContractEvent } from 'wagmi'
+
+import { PUBLIC_ALL_CHAINS } from 'src/constants/defaultChains'
+import { auctionAbi } from 'src/data/contract/abis'
+import { useCountdown, useIsMounted } from 'src/hooks'
+import { AddressType } from 'src/typings'
+
+import { AuctionPaused } from './AuctionPaused'
+import { BidActionButton } from './BidActionButton'
+import { DashboardDao } from './Dashboard'
+import {
+  auctionCardBrand,
+  bidBox,
+  daoAvatar,
+  daoAvatarBox,
+  daoTokenName,
+  outerAuctionCard,
+  stats,
+  statsBox,
+} from './dashboard.css'
+
+export const DaoAuctionCard = (
+  props: DashboardDao & { userAddress: AddressType; handleMutate: () => void }
+) => {
+  const { currentAuction, chainId, auctionAddress, handleMutate, tokenAddress } = props
+  const { name: chainName, icon: chainIcon } =
+    PUBLIC_ALL_CHAINS.find((chain) => chain.id === chainId) ?? {}
+  const router = useRouter()
+  const { endTime } = currentAuction ?? {}
+
+  const [isEnded, setIsEnded] = useState(false)
+
+  const isOver = !!endTime ? dayjs.unix(Date.now() / 1000) >= dayjs.unix(endTime) : true
+  const onEnd = () => {
+    setIsEnded(true)
+  }
+
+  useContractEvent({
+    address: auctionAddress,
+    abi: auctionAbi,
+    eventName: 'AuctionCreated',
+    chainId,
+    listener: async () => {
+      setTimeout(() => {
+        handleMutate()
+      }, 3000)
+    },
+  })
+  useContractEvent({
+    address: auctionAddress,
+    abi: auctionAbi,
+    eventName: 'AuctionBid',
+    chainId,
+    listener: async () => {
+      setTimeout(() => {
+        handleMutate()
+      }, 3000)
+    },
+  })
+  const handleSelectAuction = () => {
```

Comment:
Yeah, that's what it should be. Unfortunately wrapping this in a Link nukes the styles because of the global styles that get applied to it. Same with buttons.